src/App/Routes/HomePage/ContributionList/index.tsx

```diff
@@ -7,16 +7,16 @@ import {
   myCompletedContributionsQuery,
   myOngoingContributionsQuery,
   openedContributionsQuery,
+  accountAddressAtom,
 } from "src/state";
-import { useStarknet } from "@starknet-react/core";
 
 const ContributionListContainer: FC = () => {
-  const { account } = useStarknet();
+  const accountAddress = useRecoilValue(accountAddressAtom);
   const openContributions = useRecoilValue(openedContributionsQuery);
-  const myOngoingContributions = useRecoilValue(myOngoingContributionsQuery(account));
-  const foreignOngoingContributions = useRecoilValue(foreignOngoingContributionsQuery(account));
-  const myCompletedContributions = useRecoilValue(myCompletedContributionsQuery(account));
-  const foreignCompletedContributions = useRecoilValue(foreignCompletedContributionsQuery(account));
+  const myOngoingContributions = useRecoilValue(myOngoingContributionsQuery(accountAddress));
```

Comment:
Now that the account address is in recoil, we should change the `myOngoingContributionsQuery` to a simple selector rather than a `selectorFamily`, this would prevent having to provide the `accountAddress` as an argument. It now can be `get` in the selector `get` method :)

Reply:
Very good point, will do this